.gitignore

```diff
@@ -1,3 +1,16 @@
 .env
 node_modules/
 dist/
+
+## used for some private servers, dist/**.js concated into build
+# make shure requrie.js is not concated and main.js is concated at last
+build/
```

Comment:
'shure' => 'sure'

How to you concatenate the files?
I started miniifying with uglify, but this doesn't support ES6 yet. I started with either 'default' / 'local' grunt task for development and 'release' as a minified version where e.g. `require.js` is replaced with a dummy. Can we combine our methods somehow?